tensorboard/components/vz_example_viewer/vz-example-viewer.ts

```diff
@@ -326,29 +349,33 @@ Polymer({
         ]);
   },
 
+  selectAll: function(query: string) {
+    return d3.selectAll(
+      Polymer.dom(this.root).querySelectorAll(query) as any);
+  },
+
   haveSaliency: function() {
-    if (!this.featuresList || !this.saliency ||
+    if (!this.filteredFeaturesList || !this.saliency ||
         Object.keys(this.saliency).length === 0 || !this.colors) {
       return;
     }
 
     // TODO(jwexler): Find a way to do this without requestAnimationFrame.
```

Comment:
Just saying, this seems to happen because of `_debounceTemplate` in dom-repeat: https://github.com/Polymer/polymer/blob/1.x/src/lib/template/dom-repeat.html#L492